### DISCUSSION TITLE
Feature: *forks* (/network/members) View number of commits ahead or behind origin repo

### DISCUSSION THREAD

**Discussion by jeremyfiel** [community/community#8049] [Other Feature Feedback, Questions, & Ideas]
I would like the ability to quickly view how far ahead or behind a fork is in relation to the origin repo. I find many OSS repos have stale issues or unmerged PR's and later realize a PR pending approval was merged into a forked repo because the original maintainers did not agree with or did not maintain their work.  The ability to quickly identify which forked repo's have these types of commits would be extremely helpful. 

the page I'm referring to is `https://www.github.com/{user}/{repo}/network/members`

**Comment by ahelsley** [comment]
A similar use-case for the same requested feature:  I would use this feature when a project has silently gone stale and I want to search for more up-to-date forks to consider using instead.

**Reply by endolith** [reply]
https://addons.mozilla.org/en-US/firefox/addon/lovely-forks/ helps
